src/components/calcite-dropdown-item/calcite-dropdown-item.tsx

```diff
@@ -176,7 +176,9 @@ export class CalciteDropdownItem {
 
   @Listen("calciteDropdownItemChange", { target: "body" })
   updateActiveItemOnChange(event: CustomEvent) {
-    if (event.target === this.parentDropdownGroupEl) {
+    const parentEmittedChange = event.composedPath().includes(this.parentDropdownGroupEl);
```

Comment:
Do you know if `composedPath` is polyfilled for IE11?

I see it used other places so I'm guessing so: https://github.com/Esri/calcite-components/search?q=composedPath&unscoped_q=composedPath

Reply:
Yep, it's polyfilled. 🎉